src/gnark_backend_glue/mod.rs

```diff
@@ -1,6 +1,12 @@
+use std::os::raw::c_uchar;
+
 use acvm::{acir::circuit::Circuit, FieldElement};
 use anyhow::Result;
 
+extern "C" {
+    fn Verify() -> c_uchar;
```

Comment:
Is there a way to get a `bool` instead of a `c_uchar`?

Reply:
When you compile Go this is the resulting api:

```c
extern GoSlice Prove();
extern GoUint8 Verify();
```

And `std::os::raw` only has these types:

```rust
c_char c_schar c_uchar
c_short c_ushort
c_int c_uint
c_long c_ulong
c_longlong c_ulonglong
c_float
c_double
c_void
```

I'm not sure how to use a bool instead when importing the function